scripts/build_schemas.sh

```diff
@@ -21,14 +21,14 @@ for region_code in GB-WLS GB-ENG GB-NIR; do
                 SOURCE_FILE="source/jsonnet/northern-ireland/census_${census_type}.jsonnet"
                 ADDITIONAL_LIBRARY_PATH="source/jsonnet/northern-ireland/${census_type}/lib/"
 
-                jsonnet --tla-str region_code="${region_code}" --ext-str census_date="${CENSUS_DATE}" --jpath "${ADDITIONAL_LIBRARY_PATH}" "${SOURCE_FILE}" > "${DESTINATION_FILE}"
+                jsonnet --ext-str region_code=${region_code} --tla-str region_code="${region_code}" --ext-str census_date="${CENSUS_DATE}" --jpath "${ADDITIONAL_LIBRARY_PATH}" "${SOURCE_FILE}" > "${DESTINATION_FILE}"
```

Comment:
Given region code is now using `ext-var`, any reason to keep `tla-var`?. Or was the existing ones not updated to prevent this PR from becoming muddy?.

Reply:
I did explain the the pull request description 😉

Reply:
😄 my bad 👍